test/integration_test/applicationbackup_test.go

```diff
@@ -629,3 +700,169 @@ func commonApplicationBackupScheduleTests(
 	}
 	deletePolicyAndApplicationBackupSchedule(t, namespace, policyName, scheduleName)
 }
+
+func applicationBackupSyncControllerTest(t *testing.T) {
+	var err error
+	backupLocationName := appKey + "-backup-location-sync"
+
+	// Create myqsl app deployment
+	appCtx := createApp(t, appKey)
+
+	// Create backup location on first cluster
+	backupLocation, err := createBackupLocation(t, backupLocationName, appCtx.GetID(), storkv1.BackupLocationS3, "secret-config")
+	require.NoError(t, err, "Error creating backuplocation")
+	logrus.Infof("Created backup location:%s sync:%t", backupLocation.Name, backupLocation.Location.Sync)
+	backupLocation.Location.Sync = true
+
+	firstBackup, err := createApplicationBackupWithAnnotation(t, appKey+"-backup-sync", appCtx.GetID(), backupLocation)
+	require.NoError(t, err, "Error creating app backups")
+
+	// Create backup location on second cluster
+	err = dumpRemoteKubeConfig(remoteConfig)
+	if err != nil {
+		require.NoErrorf(t, err, "Unable to write clusterconfig: %v", err)
+	}
+
+	err = setRemoteConfig(remoteFilePath)
+	require.NoError(t, err, "Error setting remote config")
+
+	// Create namespace for the backuplocation on second cluster
+	ns, err := k8s.Instance().CreateNamespace(appCtx.GetID(),
+		map[string]string{
+			"creator": "stork-test",
+			"app":     appCtx.App.Key,
+		})
+	require.NoError(t, err, "Failed to create namespace %s: %v", appCtx.GetID(), err)
```

Comment:
No need to print err, it'll get printed if the NoError condition failed

Reply:
Fixed.